src/dawn/engine_dawn.cc

```diff
@@ -360,12 +361,27 @@ Result EngineDawn::DoClear(const ClearCommand* command) {
   color_attachment.loadOp = ::dawn::LoadOp::Clear;
   color_attachment.storeOp = ::dawn::StoreOp::Store;
 
-  // Attach that colour attachment to the render pass.
+  // Then describe the depthStencil attachment, and how it is initialized
+  // via the load ops. Both load op are "clear" to the clear values.
+  ::dawn::RenderPassDepthStencilAttachmentDescriptor depthStencil_attachment =
+      ::dawn::RenderPassDepthStencilAttachmentDescriptor();
+  depthStencil_attachment.attachment =
+      render_pipeline->depthStencil_texture.CreateDefaultView();
+  depthStencil_attachment.clearDepth = render_pipeline->clear_depth_value;
+  depthStencil_attachment.clearStencil = render_pipeline->clear_stencil_value;
+  depthStencil_attachment.depthLoadOp = ::dawn::LoadOp::Clear;
+  depthStencil_attachment.depthStoreOp = ::dawn::StoreOp::Store;
+  depthStencil_attachment.stencilLoadOp = ::dawn::LoadOp::Clear;
+  depthStencil_attachment.stencilStoreOp = ::dawn::StoreOp::Store;
+
+  // Attach the depthStencil and colour attachments to the render pass.
   ::dawn::RenderPassColorAttachmentDescriptor* rpca[] = {&color_attachment};
+  ::dawn::RenderPassDepthStencilAttachmentDescriptor* rpda = {
+      &depthStencil_attachment};
   ::dawn::RenderPassDescriptor rpd;
   rpd.colorAttachmentCount = 1;
   rpd.colorAttachments = rpca;
-  rpd.depthStencilAttachment = nullptr;
+  rpd.depthStencilAttachment = rpda;
```

Comment:
This fielld is a just a pointer.  So you don't need this intermediate rpda.  Change this to

rpd.depthStencilAttachment = &depthStencil_attachment;

Reply:
I need it because of the other change you suggested (setting depthStencil only if it is requested on amber side).